pages/docs/compute-cluster/slurm.mdx

```diff
@@ -23,7 +23,7 @@ or [let us know](/docs/compute-cluster/support-resources).
 Before we dive into the details, let's define some common terms used in SLURM:
 
 - **Login node**: A node that users log into to submit jobs to the SLURM cluster. This is where you will interact with the SLURM cluster.
-- **Compute node**: A node that runs jobs submitted to the SLURM cluster. This is where your job will run. Compute nodes are not directly accessible by users.
```

Comment:
this comment is confusing because interactive job definition clashes with this. IMO, better to leave this out because its a technical detail that doesn't affect people much, and creates confusion